### PR TITLE
Added ionice.md

### PR DESCRIPTION
**Pull request by kumon**
ionice is helpful for copying huge files, system backup and so on.

### CODE REVIEW

File: pages/common/ionice.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ionice
+
+> get/set program io scheduling class and priority
+
+- sets process with PID 89 as an idle io process
+
+`ionice -c 3 -p 89`
```
Comment (igorshubovych) [inline pages/common/ionice.md]: What do these magic numbers "2", "3" mean?

File: pages/common/ionice.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# ionice
+
+> get/set program io scheduling class and priority
```
Comment (waldyrious) [inline pages/common/ionice.md]: Suggestion: capitalize "IO", or even better, expand the acronym :)

**Comment by waldyrious** [comment]
Please replace the verbs to use the infinitive (sets --> set, runs --> run, etc.)

**Comment by kumon** [comment]
Thank you for your comments. I updated descriptions and usages.

File: pages/common/ionice.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+> Get or set program I/O scheduling class and priority.
+> Scheduling classes: 1(realtime), 2(best-effort), 3(idle).
+> Priority levels: 0(the highest) - 7(the lowest).
+
+- Set I/O scheduling class of a running process
+
+`ionice -c {{scheduling_class}} -p {{pid}}`
+
+- Run a command with altered I/O scheduling class and priority
```
Comment (waldyrious) [inline pages/common/ionice.md]: I'd say "custom" rather than "altered" here.

**Comment by waldyrious** [comment]
Much better indeed @kumon. Besides the comment I've added inline, please add spaces before the parenthesis in the main description. Other than that, looks good to go :)

**Comment by kumon** [comment]
Thank you for your review @waldyrious.
I updated it.

**Comment by rubenvereecken** [comment]
Thanks for taking care of @waldyrious' thorough comments, this looks neat. I'll merge it in.

**Comment by kumon** [comment]
Thank you so much.

**Pull request merged**